api/v1alpha1/openapi.yaml

```diff
@@ -2731,6 +2731,14 @@ components:
               description: 'The directory in which the executable will be run from if it is left empty it will run from the users home directory.'
           required:
             - run
+    DeviceSummary:
```

Comment:
Why not reuse the existing DevicesSummary?

Reply:
OK